map.js

```diff
@@ -101,9 +123,95 @@ function zoomMap(event){
     if(event.deltaY > 0) zoomLevel += 0.2;
     else zoomLevel += -0.2;
     
+    //TODO: make it so that it zooms into middle of screen rather than top left corner?
+
     //clamp zoom
     if(zoomLevel > maxZoom) zoomLevel = maxZoom;
     if(zoomLevel < minZoom) zoomLevel = minZoom;
-    console.log(zoomLevel);
     moveMap();
+}
+
+//attaches width to width of iframe
+function iFrameCheck(){
+    if(document.getElementById("iframeContainer")){
+        document.getElementById("iframeContainer").onclick = resizeMap;
+        window.removeEventListener("mousemove", iFrameCheck); //trim listener, it's served it's purpose.
+        resizeMap();
+    }
+}
+
+//matches width of map to Iframe width
+function resizeMap(){
```

Comment:
don't understand this function either

Reply:
it makes the map width equal to the iframe width.